packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/TorDropdown/index.tsx

```diff
@@ -0,0 +1,150 @@
+        position: absolute;
+        top: 8px;
+        left: 8px;
+        width: 8px;
+        height: 8px;
+        border-radius: 50%;
+        background: ${props =>
+            props.isActive ? props.theme.TYPE_GREEN : props.theme.TYPE_LIGHT_GREY};
+    }
+`;
+
+const Details = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+const Label = styled.div`
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    color: ${props => props.theme.TYPE_DARK_GREY};
+`;
+
+const Address = styled.div`
+    font-size: ${variables.FONT_SIZE.TINY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+`;
+
+const GoToTorSettings = styled.button`
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    background: none;
+    border: 0;
+    width: 16px;
+    margin: 0 10px 0 auto;
+`;
+
+interface Props {
+    marginLeft?: boolean;
+    isActive?: boolean;
+}
+
+const TorDropdown = (props: Props) => {
+    const [open, setOpen] = useState(false);
+    const [torAddress, setTorAddress] = useState('');
+    const dropdownRef = useRef<DropdownRef>();
+    const theme = useTheme();
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+    useEffect(() => {
+        window.desktopApi?.getTorAddress().then(address => setTorAddress(address));
+    });
+
+    return (
+        <Wrapper {...props}>
+            <Dropdown
+                onToggle={() => setOpen(!open)}
+                ref={dropdownRef}
+                alignMenu="right"
+                offset={34}
+                horizontalPadding={12}
+                topPadding={12}
+                bottomPadding={12}
+                minWidth={230}
+                items={[
+                    {
+                        key: 'tor',
+                        options: [
+                            {
+                                key: 'torStatus',
+                                label: (
+                                    <TorStatus>
+                                        <Indicator isActive={props.isActive} />
+                                        <Details>
+                                            <Label>
+                                                <Translation id="TR_TOR" />{' '}
+                                                {props.isActive ? (
+                                                    <Translation id="TR_ACTIVE" />
+                                                ) : (
+                                                    <Translation id="TR_DISABLED" />
+                                                )}
+                                            </Label>
+                                            {props.isActive && <Address>{torAddress}</Address>}
+                                        </Details>
+                                        <GoToTorSettings onClick={() => goto('settings-index')}>
```

Comment:
If you make the whole dropdown clickable, then we do not need onClick here.

Not your fault, but I expected that clicking on the right arrow would expand this bubble or open something there. I did not expect that it will take me to settings. I even did not notice navigating to settings (light mode).

`For the June release: It's OK if don't manage to implement the new drop down menu in Tor` 
Hmm, I am not sure what is new dropdown, if it will really change. But this one is really confusing 😢

Reply:
I agree, I made the whole thing clickable.